### DISCUSSION TITLE
ngx-editor: Fixing the position of ngx-editor-menu

### DISCUSSION THREAD

**Discussion by henafufa** [sibiraj-s/ngx-editor#486] [Q&A] [answered]
Now I need to scroll up to edit the bottom contents, how can make the ngx-editor-menu be fixed when it gets scrolled within this div?

```
  <div class="editor">
         <ngx-editor-menu [editor]="editor" [toolbar]="toolbar"> </ngx-editor-menu>
        <ngx-editor [editor]="editor" formControlName="note"> </ngx-editor>
     </div>
```
<img width="599" alt="Screen Shot 2023-07-18 at 12 15 10 PM" src="https://github.com/sibiraj-s/ngx-editor/assets/37697829/755b9580-04f2-47d9-acc4-8b7827dc5e8f">

To make it more clear, I am not referring to the floating menu
Any help would be greatly appreciated.

**Comment by henafufa** [comment]
I get it right

**Reply by sibiraj-s** [reply, marked as answer]
Glad you got it fixed.

Just as a note.
The editor and editor-menu are two different components its upto you how do you apply styles on them, align them, place them. etc.

**Comment by henafufa** [comment]
Hi Sibraj

I hope all is well with you

How to set maxlength on editor. I can't find any config variable to limit
character count

**Reply by sibiraj-s** [reply]
See https://sibiraj-s.github.io/ngx-editor/examples/reactive-forms/#validators

**Comment by henafufa** [comment]
I have tried that, Validators.maxLength(maxLength) doesn't actually work I
did provide my code

<form [formGroup]="form">
<ngx-editor
class="w-full"
[editor]="editor"
formControlName="editorContent"
[placeholder]="'Add notes here...'"
>
</ngx-editor>
</form>

form = new FormGroup({
editorContent: new FormControl(null, [Validators.required(), Validators.
maxLength(500)]),
});

**Comment by henafufa** [comment]
https://stackblitz.com/edit/ngx-editor-2fzuev?file=src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html
